smithy-swift-codegen-test/build.gradle.kts

```diff
@@ -0,0 +1,67 @@
+/*
```

Comment:
Q: So does the SDK get generated & tests get run against generated SDK all as part of the gradle build command?

Reply:
Currently the gradle build command only generates the swift SDK and tests code into the `Sources` and `Tests` directories. To run the tests, we run `swift test` as usual.